### PR TITLE
BSatishSutar-patch-1 - Added Social links and milestones

### PR DESCRIPTION
**Pull request by BSatishSutar**
## Check List (Check all the applicable boxes) 

- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I have added my social links and milestones

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5196"></a>

### CODE REVIEW

**Comment by BSatishSutar** [comment]
Modified BSatishSutar.json file as advised by Pradumna

File: data/BSatishSutar.json [hunk 1]
```diff
@@ -1,37 +1,95 @@
...
+			"date": "Dec 2022",
+			"icon": "SiBeehiiv",
+			"color": "blue",
+			"description": "Cloud and DevOps",
+			"url": "https://cloud101-newsletter-09c659.beehiiv.com/"
+		}
+	]
+
+
+}
```
Comment (Panquesito7) [inline data/BSatishSutar.json:95]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/BSatishSutar.json (hunk 1 above)
Comment (BSatishSutar) [reply data/BSatishSutar.json:95]: ok. Let me work on it.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by amandamartin-dev — APPROVED** [review]

**Pull request merged**